Cargo.lock

```diff
@@ -1780,6 +1780,7 @@ dependencies = [
  "jsonrpc-http-server",
  "lazy_static",
  "once_cell",
+ "openssl-sys",
```

Comment:
Hmm, perhaps this happened in the middle of the merge from `main`. Can you revert the changes to `Cargo.lock`? Everything else looks good 👍

Reply:
Oh wait, I'm seeing the same thing locally... I think we missed something in the latest PR to `main`

Reply:
K, leave as is. This was a mistake with previous PR.